package.json

```
@@ -22,10 +22,11 @@
     "test-crossbrowser": "au e2eCrossbrowser",
     "lint": "eslint \"test/**/*.ts\" \"src/**/*.ts\" \"electron_app/**/*.js\" \"bin/**/*.js\" \"aurelia_project/**/*.js\" \"aurelia_project/**/*.ts\"",
     "lint-fix": "eslint --fix \"test/**/*.ts\" \"src/**/*.ts\" \"electron_app/**/*.js\" \"bin/**/*.js\" \"aurelia_project/**/*.js\" \"aurelia_project/**/*.ts\"",
+    "electron-build-windows": "npm run electron-prepare && build --config build/electron-builder.yml --publish never --windows; npm run electron-cleanup",
+    "electron-build-macos": "npm run electron-prepare && build --config build/electron-builder.yml --publish never --macos; npm run electron-cleanup",
+    "electron-cleanup": "node electron-builder/reset-name-in-package-json.js",
+    "electron-prepare": "node electron-builder/set-name-in-package-json.js && cat package.json && electron-rebuild && npm run build && node electron-builder/prepare-electron-build.js",
```

```suggestion
    "electron-prepare": "node electron-builder/set-name-in-package-json.js && electron-rebuild && npm run build && node electron-builder/prepare-electron-build.js",
```

Könnte wahrscheinlich entfernt werden, oder soll das extra ausgegeben werden?